packages/neuron-ui/src/components/TransactionFeePanel/index.tsx

```diff
@@ -1,81 +1,143 @@
-import React, { useState } from 'react'
-import { Stack, Toggle, IDropdownOption } from 'office-ui-fabric-react'
-import Dropdown from 'widgets/Dropdown'
+import React, { useState, useMemo, useRef } from 'react'
+import clsx from 'clsx'
+import { Stack } from 'office-ui-fabric-react'
```

Comment:
Avoid using `official ui fabric react` because it's being removed from the project

We're trying to remove all unnecessary external deps, as https://github.com/Magickbase/neuron-public-issues/issues/66 and https://github.com/Magickbase/neuron-public-issues/issues/68

Reply:
BTW, we can use `grid` because the runtime is stable and `grid` is supported